modules/aruco/tutorials/aruco_board_detection/aruco_board_detection.markdown

```diff
@@ -55,22 +69,35 @@ In fact, to use marker boards, a standard marker detection should be done before
 The aruco module provides a specific function, ```estimatePoseBoard()```, to perform pose estimation for boards:
```

Comment:
It's old API. New APi does not provide pose estimation. you need to replace the text and code with solvePnP call.